src/fxcorrelator_xengops.py

```diff
@@ -23,6 +23,9 @@ def __init__(self, corr_obj):
         self.hosts = corr_obj.xhosts
         self.logger = corr_obj.logger
         # do config things
+        self.xengines_per_xhost = int(self.corr.configd['xengine']['x_per_fpga'])
+        self.channels_per_xhost = self.corr.n_chans / len(self.hosts)
+        self.channels_per_xengine = self.channels_per_xhost / self.xengines_per_xhost
```

Comment:
Just for insterest's sake: are these numbers always going to be neatly divisible? Some powers of 2 thing going on?